src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+const FEMALE_SIGN = 'f';
+
+const NOT_MARRIED_TEXT = 'I am not married';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const partnerStatus = sex === FEMALE_SIGN
+    ? 'husband'
+    : 'wife';
+  const marriedStatusContent = isMarried
+    ? `${partnerName} is my ${partnerStatus}`
+    : NOT_MARRIED_TEXT;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {
+        age && (
```

Comment:
You have to explicitly state that `age` it's a boolean, because it can just render 0.